live/kafka/main.tf

```diff
@@ -0,0 +1,87 @@
+resource "helm_release" "kafka_operator" {
+  name       = var.name
+  repository = var.repository
+  chart      = var.chart
+  version    = var.chart_version
+
+}
+
+resource "kubectl_manifest" "kafka" {
+  yaml_body = <<YAML
```

Comment:
Using kubectl provider and `yaml_body` we have the whole manifest as a string.
What do you think about using the `kubernetes_manifest` from kubernetes provider?
We could use HCL and add some variables here (for example replication, kafka version etc).
On the other hand in the documentation of `kubernetes_manifest`:
> This resource requires API access during planning time. This means the cluster has to be accessible at plan time and thus cannot be created in the same apply operation. We recommend only using this resource for custom resources or resources not yet fully supported by the provider.

And this can be problematic when running plan on "empty" env.
Maybe we could use kubectl provider and still have some variables?